scripts/lambdaRun.py

```diff
@@ -57,7 +53,9 @@ def main():
 
     elif re.match(r'^build-(?:development|qa|production)', runType):
```

Comment:
the if/elif plus the regexes seems a little multi-dimensional and hard to debug if there's a subtle problem in a regex. With one exception, the same code is getting run for each, just with some different variables. I can think of ways to do it with a dict where the values are functions to run, then with something like:

    funcMap.get(runType, errFunc)()

You can run the function matching the runType, or a function to log an error if nothing matches. The advantage would be that the mapping would contain all the explicitly valid runTypes and you don't have to pick your way through elifs and regexes.

Reply:
That's an excellent idea. I haven't really thought too much about this function/file since the beginning because it hasn't posed any issues, but that does seem like a DRY-er and cleaner implementation. anything that eliminates regexes is generally 👍 in my book.

Will take a stab at implementing something along these lines.